dev/data/dashboard.json

```diff
@@ -3224,7 +3224,7 @@
             ]
           }
         },
-        "bottomlist-1": {
+        "rightlist": {
```

Comment:
Renaming these to be clearer at what they are, so it was easier to modify the values.